recipes/configs/qwen2/1.5B_lora.yaml

```diff
@@ -23,9 +23,8 @@ model:
   _component_: torchtune.models.qwen2.lora_qwen2_0_5b
```

Comment:
i am misreading it? The config name says 1.5, but the component and instructions are 0.5

I personally dont like having "." in the middle of a file name. I wonder if it should be 1_5B

Reply:
good catch! I hadn't tested distributed. I'm not married to the config name but I agree with you